InventoryInStorePickupFrontend/Block/Checkout/Onepage/Success.php

```diff
@@ -0,0 +1,59 @@
+<?php
```

Comment:
Please, use view model pattern to inject new behavior instead of block override.

Reply:
Done.